contracts/bridge/contracts/interfaces/ILightClient.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.0 <0.7.0;
+
+interface ILightClient {
+    function verify_messages_proof(
```

Comment:
```suggestion
    function verify_lane_data_proof(
```

Rename to avoid duplicate with apis in `function receive_messages_proof` and `function receive_messages_delivery_proof(`